lcls-twincat-motion/Library/Version/Global_Version.TcGVL

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+<TcPlcObject Version="1.1.0.1">
   <GVL Name="Global_Version" Id="{e584f020-617f-4164-a737-6e483df3aa38}">
     <Declaration><![CDATA[{attribute 'TcGenerated'}
 {attribute 'no-analysis'}
 {attribute 'linkalways'}
 // This function has been automatically generated from the project information.
 VAR_GLOBAL CONSTANT
     {attribute 'const_non_replaced'}
-    stLibVersion_lcls_twincat_motion : ST_LibVersion := (iMajor := 0, iMinor := 0, iBuild := 0, iRevision := 0, sVersion := '0.0.0');
+    stLibVersion_lcls_twincat_motion_zlentz : ST_LibVersion := (iMajor := 0, iMinor := 0, iBuild := 0, iRevision := 0, nFlags := 0, sVersion := '0.0.0');
```

Comment:
I've actually already fixed this in https://github.com/pcdshub/tc-release/pull/33

Reply:
Ah, I had looked at https://github.com/pcdshub/tc-release/blob/master/tc_release/tcgvl.txt and saw it there still. Didn't realize it was handled in the code portion.

Reply:
Actually you're right right about this one, I must be producing a file with a stale product version in these cases. It doesn't cause any harm but it could potentially be confusing (it confused both of us just now).